packages/server/lib/project.js

```diff
@@ -158,6 +158,7 @@ class Project extends EE {
 
         options.onError(err)
       },
+      onWarning: options.onWarning,
```

Comment:
add this as an empty default function here: https://github.com/cypress-io/cypress/blob/deprecated-plugins-before-browser-launch/packages/server/lib/project.js#L69

Reply:
although really this is just a poor mans non-typescript definition, so probably would just be better to handle this in typescript